catalog/ui/catalog-ui-metacard-sharing-impl/src/main/java/org/codice/ddf/catalog/ui/sharing/ShareableMetacardImpl.java

```diff
@@ -55,30 +52,27 @@ protected ShareableMetacardImpl(Metacard metacard) {
   }
 
   @VisibleForTesting
-  ShareableMetacardImpl() {
+  public ShareableMetacardImpl() {
     super(
         new MetacardTypeImpl(
             "metacard.sharing", Arrays.asList(new CoreAttributes(), new SecurityAttributes())));
   }
 
   @VisibleForTesting
-  ShareableMetacardImpl(String id) {
+  public ShareableMetacardImpl(String id) {
     this();
     setId(id);
   }
 
   /**
-   * Check if a given metacard is a shareable metacard by checking the presence of security
+   * Check if a given metacard is a sharing metacard by checking the presence of security
    * attributes.
    *
    * @param metacard the metacard to check.
-   * @return true if the provided metacard is a shareable metacard, false otherwise.
+   * @return true if the provided metacard is a sharing metacard, false otherwise.
    */
   public static boolean canShare(Metacard metacard) {
     return metacard != null
-        && (WorkspaceMetacardImpl.isWorkspaceMetacard(metacard)
-            || QueryTemplateMetacard.isQueryTemplateMetacard(metacard)
-            || AttributeGroupMetacard.isAttributeGroupMetacard(metacard))
```

Comment:
❗️ Woah. Was this removed by accident? I'm almost certain resource metacards will now get processed by this class, which is not what we want.